internal/server/server.go

```diff
@@ -76,14 +78,42 @@ func (s *SOMASServer) logf(format string, a ...interface{}) {
 }
 
 func initEnvironment() *disasters.Environment {
+	var cp = disasters.Commonpool{900, 1000}
 	_, clientMap := getClientInfosAndMapFromRegisteredClients(baseclient.RegisteredClients)
 	islandIDs := make([]shared.ClientID, 0, len(clientMap))
 	for id := range clientMap {
 		islandIDs = append(islandIDs, id)
 	}
 	xBounds := [2]float64{0, 10}
 	yBounds := [2]float64{0, 10}
-	dp := disasters.DisasterParameters{GlobalProb: 0.1, SpatialPDF: "uniform", MagnitudeLambda: 1.0}
-	env, _ := disasters.InitEnvironment(islandIDs, xBounds, yBounds, dp)
+	dp := disasters.DisasterParameters{GlobalProb: 1, SpatialPDF: "uniform", MagnitudeLambda: 1.0}
+	env, _ := disasters.InitEnvironment(islandIDs, xBounds, yBounds, dp, cp)
 	return env
 }
+
+//islandDistribute distributes leftover resource after disaster's mitigation
+func (s *SOMASServer) islandDistribute(leftover uint, disasterReport string) { //receives 
+	if disasterReport != "No disaster reported." {
+		var clientPortion int
+		clientPortion = int(leftover / 6) // equally divide leftover
+		for _, id := range shared.TeamIDs {
+		ci := s.gameState.ClientInfos[id]
+		ci.Resources = ci.Resources + clientPortion //adds to each client its portion
+		s.gameState.ClientInfos[id] = ci
+		s.gameState.Environment.CommonPool.Resource = 0;
+	}
+		
+	}
+	
+
+}

```

Comment:
Fix indentation here? :D

Reply:
logic is the following: leftovers after mitigation shared to islands, zero  remains in common pool

Reply:
yep :) the curly brace indentation thing threw me off a bit, just need to fix that